enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,297 @@
+released to production one release after the other components achieve **GA**.  Once this milestone is realized, the **log-view-plugin** will
+no longer rely upon ViaQ attributes which may break existing log queries.
+
+#### Log Collection and Forwarding
+
+* The **cluster-logging-operator** will be updated to evaluate the OTLP annotation:
+    * Absence of the annotation implies forwarding to **LokiStacks** using ViaQ data model
+    * Existence of the annotation, the operator:
+        * will validate the list of names maps to a defined, **LokiStack** output and rejects the **ClusterLogForwarder** for any mismatches
+        * will internally initialize OTLP outputs for the enabled **LokiStacks**, in lieu of Loki outputs
+        * will configure **LokiStack** OTLP outputs to include MVP labels
+
+Once this enhancement achieves General Acceptance, OTEL will be the default data model when forwarding logs to **LokiStack**.  Cluster administrators will
+be able to support the Tech Preview behavior until it is removed by annotating the **ClusterLogForwarder** like the following:
+
+```
+   logstore.observability.openshift.io/viaq: [[LOKISTACK_NAMES]]
+```
+* LOKISTACK_NAMES := Comma delimited list of **clusterlogforwarder.spec.output[].name** of type **lokiStack**
+
+
+* The **cluster-logging-operator** will be updated to evaluate the ViaQ annotation:
+    * Absence of the annotation implies forwarding to **LokiStacks** using OTEL data model
+    * Existance of the annotation, the operator:
+        * will validate the list of names maps to a defined, **LokiStack** output and rejects the **ClusterLogForwarder** for any mismatches
+        * will internally initialize Loki outputs for the enabled **LokiStacks**, in lieu of OTLP outputs
+        * will configure **LokiStack** LokiStack outputs to include OTEL equivalent MVP labels
+
+52
+### Risks and Mitigations
+
+### Drawbacks
+
+This design offers a transitional plan to allow users to change the enrichment model of their log storage while continuing to use the console web UI plugin and
+MVP attributes.  Advanced queries using [logQL](https://grafana.com/docs/loki/latest/query/) that rely upon attributes other then MVP may no longer function.
+
+
+## Open Questions [optional]
+
+1. Do we need to forward the complimentry UI OTEL attributes when configured for ViaQ?
+
+## Test Plan
+
+## Graduation Criteria
+
+### Dev Preview -> Tech Preview
+
+- Ability to conditionally configure the datamodel(ViaQ or OTEL) when forwarding to in-cluster LokiStack
+- Ability to retrieve OTEL formatted log entries from in-cluster LokiStack using ViaQ attributes: log_type, kubernetes_namespace_name, kubernetes_container_name, kubernetes_pod_name
+- Ability to use existing log based alerts after logging upgrade without changes
+- Announce deprecation of log forwarding to LokiStack using ViaQ data model, to be removed **two** logging releases after GA
```

Comment:
With the move to OTLP and away from ViaQ for tenant `openshift-logging` in a sense, we are also deprecating `schemas` < v13 correct? Do we need to "prepare" for this? We already have an alert informing users to update but maybe for 6.1 we should tune the alert, WDYT? cc @periklis @xperimental

Reply:
We probably should update the message on the existing alert, so that it says something like "OTLP will not work correctly before v13". Interestingly, it seems to just ignore the `structured_metadata` when ingested over OTLP with an older schema version instead of returning an error as with the Push-API.

Reply:
Impl detail from LokiStack not requiring changes to this document. Resolving comment